packages/react-components/src/components/gif.tsx

```diff
@@ -277,15 +228,20 @@ const Gif = ({
             onContextMenu={(e: SyntheticEvent<HTMLElement, Event>) => onGifRightClick(gif, e)}
         >
             <div style={{ width, height, position: 'relative' }} ref={container}>
-                <img
-                    className={Gif.imgClassName}
-                    src={showGif ? fit : placeholder}
-                    style={{ background }}
-                    width={width}
-                    height={height}
-                    alt={getAltText(gif)}
-                    onLoad={showGif ? onImageLoad : () => {}}
-                />
+                <picture>
+                    <source type="image/webp" srcSet={rendition.webp} />
```

Comment:
`webp` detection was impossible on the server, so we didn't know what image to render in SSR. This removes our custom detection and solves the SSR problem.